### PR TITLE
task1/html: links

### PR DESCRIPTION
**Pull request by qnislavnachev**
there is a 4 links that should open local text file, directory, html file and
website in a new tab

### CODE REVIEW

File: links.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<body>
+<a href="file:///D:/Games" target="_blank">Click here to browse directory</a><br><br>
+<a href="file:///C:/Users/IaNiTyy/Desktop/parfum.txt" target="_blank">Click here to browse a txt file</a><br><br>
+<a href="file:///C:/Users/IaNiTyy/Desktop/links.html" target="_blank">Click here to browse html file</a><br><br>
```
Comment (tsony-tsonev) [inline links.html]: we should always use relative paths to files in the same dir from the  source html or in inner dirs relative to the html

**Comment by qnislavnachev** [comment]
I ve try to do it with relative path but on my PC i am still wndows and doesnt recognize my files, it need absolute path like this

**Comment by tsony-tsonev** [comment]
Almost every site works that way and they all work on windows.

**Comment by qnislavnachev** [comment]
I will try to fix it

**Comment by tsony-tsonev** [comment]
What was your unsuccessful approach?

**Comment by qnislavnachev** [comment]
well I think that is not unsuccessfull approach but I`ll check that just for information

**Comment by tsony-tsonev** [comment]
I meant how did you try to use the relative path, cause you sad that it did not worked.

**Comment by qnislavnachev** [comment]
I ve tried like this "/file.txt", "file:///file.txt" and similar to this

**Comment by tsony-tsonev** [comment]
try this one
```html
<a href="test.txt" target="_blank">link</a>
```
the test.txt should be in the same dir as the html

**Comment by qnislavnachev** [comment]
yeah that works thank you, now i am feeling stupid :D

**Comment by tsony-tsonev** [comment]
no worries, it is always the last option you do not try that is the working one.

**Pull request merged**